build/ci/phase-template.yml

```diff
@@ -33,13 +33,13 @@ phases:
           ${{ insert }}: ${{ parameters.customMatrixes }}
       ${{ insert }}: ${{ parameters.queue }}
     steps:
+    - ${{ if eq(parameters.name, 'MacOS') }}:
+      - script: brew update && brew install libomp && brew install mono-libgdiplus gettext && brew link gettext --force && brew link libomp --force
```

Comment:
>libomp [](start = 44, length = 6)

do we still need to install libomp if we install it in vsts-ci.yml?

Reply:
I believe it does as Eric's comment earlier was to make sure to also add this to vsts-ci.yml. My impression is that the vsts-ci.yml is for the gated build (post merge) and vsts-ci.yml is separate from phase-template.yml, but its a good question. @eerhardt can you clarify? Thanks!

Reply:
they are 2 completely separate builds:

1. For PR validation and post merge CI.
2. For official build - which signs assemblies, publishes nuget packages and symbols, etc.